_docs_integrate/new-versions.md

```diff
@@ -0,0 +1,150 @@
+---
+# Start automatic generation
+permalink: integrate/new-versions
+published: true
+title: "New versions"
+type: scenario
+toc: true
+properties:
+  - id: SC115
+  - category: Data Model
+  - description:
+  - customer:
+  - component: integrate
+  - level:
+  - implementation status: DOCS ONLY
+  - documentation status:
+  - published:
+  - link: new-versions
+require:
+required_by:
+# End automatic generation
+---
+
+# Migration guide due to release/v5
+
+After merging the PR release/v5 into the main branch of the runtime, there are some backwards incompatible changes about which the Integrators will be informed transparently in the following.
```

Comment:
I'd leave out this completely. When we merge the v5 branch of the docs we will not need the paragraph anymore.